code/src/index.css

```diff
@@ -1,13 +1,199 @@
+*{
+  margin: 0px;
+  padding: 0px;
+}
+
 body {
   margin: 0;
   font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen",
     "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue",
     sans-serif;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
+  background-color: black;
 }
 
 code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
     monospace;
 }
+
+.body{
+  position: relative;
+  padding: 10px 30px 10px 30px;
+  margin: 10px auto;
+}
+
+header{
+  color: white;
+  font-size: 15px;
+  padding: 10px 0px;
+  margin: 50px 0px 10px 45px;
+  border-bottom: solid white 1px;
+}
+
+.album-wrapper{
+  display: grid;
+  justify-content: center;
+}
+
+.album-card{
+  display: grid;
+  justify-content: center;
+}
+
+.text-wrapper{
+  margin:20px 0px;
+  width: 300px;
+}
+
+.artist-name{
+margin: 15px 0px;
+}
+
+
+.album-title{
+  font-size: 14px;
+  font-family: Helvetica, sans-serif;
+  color: white; 
+}
+
+.artist-name{
+  font-size: 14px;
+  font-family: Helvetica, sans-serif;
+  color: #a0a0a0;
+}
+
+a {
+  text-decoration: none;
+  color: #a0a0a0;
+}
+
+
+
+/* HOVER EFFECTS */
+a:hover {
+  text-decoration: underline;
+  transition: 0.5s ease;
+  color: #fff;
+}
```

Comment:
Not sure about this, but we THINK that the hover effect only should be in the media query for desktop. It can be "laggigt" and you don't hover with your finger anyway.